fastapi/applications.py

```diff
@@ -152,55 +149,6 @@ def __init__(
         self.middleware_stack: Union[ASGIApp, None] = None
         self.setup()
 
-    def build_middleware_stack(self) -> ASGIApp:
-        # Duplicate/override from Starlette to add AsyncExitStackMiddleware
-        # inside of ExceptionMiddleware, inside of custom user middlewares
-        debug = self.debug
-        error_handler = None
-        exception_handlers = {}
```

Comment:
We can get rid of all of this now because the structure is `Starlette -> ServerErrorMiddleware -> User middleware -> routing -> HTTPException handling -> AsyncExitStack -> Dependencies -> Endpoint` (in other words, Starlette moved what used to be ExceptionMiddleware to be inside of routing)